ecl2df/common.py

```diff
@@ -621,3 +622,47 @@ def stack_on_colnames(dframe, sep="@", stackcolname="DATE", inplace=True):
     del dframe["level_0"]
     dframe.index.name = ""
     return dframe
+
+def read_zonemap(filename):
+    """Return a dictionary from (int) K layers in the simgrid to strings
+
+        Typical usage is to map from grid layer to zone names.
+
+        The layer filename must currently follow format::
+
+          'ZoneA' 1-4
+          'ZoneB' 5-10
+
+        where the single quotes are optional for zones without spaces.
+        Write single layer zones as 11-11. NB: ResInsight requires single
+        quotes always.
+
+        Args:
+            filename (str): Absolute path to a zone map file (lyr format)
+
+        Returns:
+            dict, integer keys which are the K layers. Every layer mentioned
+                in the interval in the input file is present. Can be empty.
+    """
+    assert isinstance(filename, str)
+    with open(filename, "r") as fin:
```

Comment:
After learning `pathlib` I prefer `Path(filename).read_text().splitlines()`

Reply:
(that would also make it possible to supply `Path` objects to the function)

Reply:
That might also solve the error for the tests to pass.

Reply:
Are you thinking that I should make a Path object an input parameter to the function? F.ex optional or instead of the string?